spec/services/booked_visits_csv_exporter_spec.rb

```diff
@@ -40,6 +40,7 @@
       expect(row['Prisoner number']).to eq(booked_visit.prisoner_number)
       expect(row['Slot granted']).
         to eq(instance.format_slot_for_staff(booked_visit.slot_granted))
+      expect(row['Closed visit']).to be_nil
```

Comment:
If it's not a closed visit it doesn't return false, just a nil value. An error is return if I use:
expect(row['Closed visit']).to eq('false')